spec/serializers/web_hook_user_serializer_spec.rb

```diff
@@ -42,6 +42,7 @@
       :recent_time_read,
       :primary_group_id,
       :primary_group_name,
+      :flair_group_id,
```

Comment:
I think we should leave this out of the user web hook until a user specifically requests for it.